### PR TITLE
 Fix fireplace multi-search for FxOS Add-ons (bug 1210029)

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

File: mkt/search/serializers.py [hunk 1]
```diff
@@ -141,6 +141,7 @@ def to_native(self, obj):
 
         serializer = self.serializers.get(doc_type)
         if serializer is None:
+            import debug
```
Comment (chuckharmston) [inline mkt/search/serializers.py]: 😒

Comment (diox) [inline mkt/search/serializers.py]: lol

Comment (diox) [inline mkt/search/serializers.py]: interestingly I don't think this branch is ever hit :) I removed it, thanks

**Comment by chuckharmston** [comment]
Better commit message? Should be clear what is being fixed.

**Comment by chuckharmston** [comment]
r+

**Pull request merged**